src/libraries/System.Net.Security/src/System/Net/Security/SslStreamCertificateContext.Windows.cs

```diff
@@ -16,7 +16,7 @@ internal static SslStreamCertificateContext Create(X509Certificate2 target)
             return new SslStreamCertificateContext(target, Array.Empty<X509Certificate2>());
         }
 
-        private SslStreamCertificateContext(X509Certificate2 target, X509Certificate2[] intermediates)
+        private SslStreamCertificateContext(X509Certificate2 target, X509Certificate2[] intermediates, SslCertificateTrust? trust = null)
```

Comment:
We probably want the param defaults to be the same across platforms.

Reply:
fixed. We don't really need defaults on private constructor.